boards/arm/b_u585i_iot02a/doc/index.rst

```diff
@@ -175,7 +175,7 @@ The Zephyr b_u585i_iot02a board configuration supports the following hardware fe
 +-----------+------------+-------------------------------------+
 | GPIO      | on-chip    | gpio                                |
 +-----------+------------+-------------------------------------+
-| I2C       | on-chip    | i2c                                |
+| I2C       | on-chip    | i2c                                 |
```

Comment:
This change needs to move to commit `boards: b_u585i_iot02a: Enable i2c and hts221 mems sensors`

Reply:
Good catch, txs